azure-pipelines.yml

```diff
@@ -99,14 +95,13 @@ stages:
               TargetFolder: '$(Build.ArtifactStagingDirectory)'
           - task: PublishBuildArtifacts@1
             displayName: 'Publish Artifact: drop'
+
   - stage: Sandbox
     condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
     jobs:
-      - template: devops-templates/terraform-apply-template.yaml
-        parameters:
-          environment: 'sbox'
-          agentPool: ${{ variables.agentPool }}
-          terraformVersion: ${{ variables.terraformVersion }}
+      - job: Dummy
```

Comment:
why not remove the stage?

Reply:
I could remove the stage completely but all other steps have been set with a dependency on it.

Reply:
Done!